scrcpy/src/main/java/io/github/miuzarte/scrcpyforandroid/widgets/DeviceWidgets.kt

```
@@ -0,0 +1,1194 @@
+                    .padding(horizontal = UiSpacing.CardContent, vertical = UiSpacing.Medium),
+            ) {
+                Column(verticalArrangement = Arrangement.spacedBy(UiSpacing.Tiny)) {
+                    Text(
+                        text = "分辨率: ${session.width}x${session.height}",
+                        color = Color.White,
+                        fontSize = 13.sp,
+                        fontWeight = FontWeight.SemiBold,
+                    )
+                    @SuppressLint("DefaultLocale")
+                    Text(
+                        text = "FPS: ${String.format("%.1f", currentFps.coerceAtLeast(0f))}",
```

detekt (1.23.8)

[warning] String.format("%.1f", currentFps.coerceAtLeast(0f)) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)